CONTRIBUTION.md

```diff
@@ -0,0 +1,174 @@
+Apache Hadoop Ozone Contribution guideline
+===
+
+Ozone is a part of the Apache Hadoop project. The bug tracking system for Ozone is under the [Apache Jira project named HDDS](https://issues.apache.org/jira/projects/HDDS/).
+
+If you are familiar with contributing to Apache Hadoop, then you already know everything you need to know to start filing Ozone bugs and submitting patches.
+
+If you have never contributed to Apache Hadoop before, then you may find it useful to read [How To Contribute](https://cwiki.apache.org/confluence/display/HADOOP/How+To+Contribute+to+Ozone).
+
+This document summarize the contribution process and defines the differenceis.  
+
+## What can I contribute?
+
+We welcome contributions of:
+
+ * **Code**. File a bug and submit a patch, or pick up any one of the unassigned Jiras.
+   * [Newbie Ozone jiras](https://s.apache.org/OzoneNewbieJiras)
+   * [All open and unassigned Ozone jiras](https://s.apache.org/OzoneUnassignedJiras)
+ * **Documentation Improvements**: You can submit improvements to either:
+     * Ozone website. Instructions are here: [Modifying the Ozone Website](https://cwiki.apache.org/confluence/display/HADOOP/Modifying+the+Ozone+Website)
+     * Developer docs. These are markdown files [checked into the Apache Hadoop Source tree](https://github.com/apache/hadoop-ozone/tree/master/hadoop-hdds/docs/content).
+ * The [wiki pages](https://cwiki.apache.org/confluence/display/HADOOP/Ozone+Contributor+Guide): Please contact us at hdfs-dev@hadoop.apache.org and we can provide you write access to the wiki.
+ * **Testing**: We always need help to improve our testing
+      * Unit Tests (JUnit / Java)
+      * Acceptance Tests (docker + robot framework)
+      * Blockade tests (python + blockade) 
+      * Performance: We have multiple type of load generator / benchmark tools (`ozone freon`, `ozone genesis`). Which can be used to test cluster and report problems.
+ * **Bug reports** pointing out broken functionality, docs, or suggestions for improvements are always welcome!
+ 
+## Who To Contact
+
+If you have any questions, please don't hesitate to contact
+
+ * in **mail**: use hdfs-dev@hadoop.apache.org.
+ * in **chat**: You can find the #ozone channel at the ASF slack. Invite link is [here](http://s.apache.org/slack-invite) 
+ * **meeting**: [We have weekly meetings](https://cwiki.apache.org/confluence/display/HADOOP/Ozone+Community+Calls) which is open to anybody. Feel free to join and ask any questions
+    
+## Building from the source code
+
+### Requirements
+
+Requirements to compile the code:
+
+* Unix System
+* JDK 1.8
+* Maven 3.5 or later
+* Protocol Buffers 2.5
```

Comment:
This, let us move away. Almost impossible to find the downloads these days, and even if you do, i think you have compile yourself.

Reply:
@elek Thoughts on this?

Reply:
Ok, someone just hit this issue and I have left comments on how to install it. Perhaps we need to have  dedicated section on how to install protocol buffers 2.5, since we have to download source, build and install it. Do you think we can build a tarball for Linux and macOs and post it in our wiki ? With *HUGE* disclaimer that we are just being helpful and this is not a release binary at all ?

Reply:
@anuengineer You read my mind. I think for now, we can proceed and merge this PR. This can be followed up in another Jira. Thoughts ?

Reply:
Agree. Let's describe the current situation here and update protobuf soon. I added a simple section about downloading + compiling protobuf